certified-connectors/USB4SAP/apiDefinition.swagger.json

```diff
@@ -0,0 +1,71 @@
+{ 
+    "swagger": "2.0", 
+    "info": { 
+        "title": "USB4SAP", 
+        "version": "1.0", 
+        "description": "Universal Synch Bridge for SAP: One access point for all SAP integration, analytics and UI needs. Simplify modern analytics, apps and automations. Provision SAP data, insights and automation On-demand.", 
+        "contact": { 
+            "name": "Ecoservity", 
+            "url": "https://www.ecoservity.com/#contact", 
+            "email": "hello@ecoservity.com" 
+        } 
+    }, 
+    "host": "https://ecs-apim1.azure-api.net/usb4sap_live_cache-v2/", 
+    "paths": { 
+        "/sap/opu/odata/ECOS/OBJ2CLOUD_V2_SRV/ET_DatasetSet": { 
+            "GET": { 
+                "summary": "GET call to extract metadata & data from SAP data objects", 
+                "operationId": "GetCallExtractMetadata", 
+                "description": "The GET call extracts data alongwith its metadata from SAP, based on the selection criteria by user or background scheduled job", 
+                "parameters": [ 
+                    { 
+                        "name": "$filter", 
+                        "in": "query", 
+                        "required": true, 
+                        "type": "string", 
+                        "x-example": "ObjectName eq 'T001W' and ObjectType eq 'T' and Delimiter eq '|' and ImportGeneric eq 'ALVV-MG' and DataDest eq '0:0|' " 
+                    }, 
+                    { 
+                        "name": "$format", 
+                        "in": "query", 
+                        "required": false, 
+                        "type": "string", 
+                        "x-example": "json" 
+                    } 
+                ], 
+                "responses": { 
+                    "200": { 
+                        "description": "OK" 
+                    }, 
+                    "400": { 
+                        "description": "Bad Request" 
+                    }, 
+                    "404": { 
+                        "description": "Not Found" 
+                    }, 
+                    "500": { 
+                        "description": "Application Error" 
+                    }, 
+                    "default": { 
+                        "description": "Swagger generated by Ecoservity" 
+                    } 
+                } 
+            } 
+        }, 
+        "definitions": {} 
+    }, 
+    "x-ms-connector-metadata": [ 
+        { 
+            "propertyName": "Website", 
+            "propertyValue": "www.ecoservity.com" 
+        }, 
+        { 
+            "propertyName": "Privacy policy", 
+            "propertyValue": "NA" 
+        }, 
+        { 
+            "propertyName": "Categories", 
+            "propertyValue": "B2B Analytics" 
```

Comment:
@ecsadm this is not a valid category. you can choose one of the below.
Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]